src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -200,7 +201,7 @@ public function get_grid_days( $date = null, $force = false ) {
 			$start_meta_key = '_EventStartDate';
 			$end_meta_key   = '_EventEndDate';
 
-			if ( Timezones::is_mode( 'site' ) ) {
+			if ( $use_site_timezone ) {
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 4623146e273108437a1e5d29b87f5c11</sub>